README.md

```diff
@@ -51,15 +51,30 @@ cargo install --git https://github.com/paritytech/ink cargo-contract
 
 Add `--force` option to update to the most recent `cargo-contract` version.
 
-Please visit [the documentation](https://substrate.dev/substrate-contracts-workshop/#/0/building-your-contract)
-about building contracts and generating metadata.
+### Build example contract and generate the contracts ABI
 
-### Hello, World! - The Flipper
+To build a single example and generate the contracts wasm file, navigate to the root of the example smart contract and run:
+
+`cargo contract build`
+
+To generate the Application Binary Interface (ABI) of the contract, run the following command:
```

Comment:
```suggestion
To generate the Application Binary Interface (ABI), or metadata, of the contract, run the following command:
```

Reply:
I think there are too many commas here, a native speaker should check the punctuation. // cc @shawntabrizi @ascjones

Reply:
@Stefie Robin expressed to be that the term "ABI" is quite a poor description of the file. In general we should refer to it as the contract metadata.

If I were to write the sentence with both I would say:

> To generate the contract metadata (a.k.a. the contract ABI), run the following command:

In this case, I do not write out ABI, cause either they know it already, or they don't care about "a.k.a."

Reply:
Hmm, ok. I copied the wording from the workshop https://substrate.dev/substrate-contracts-workshop/#/0/building-your-contract, though. It should be changed there, too, then.

But I don't fully agree because `ABI` is kind of a fixed term that is used everywhere in the Polkadot-JS repos (https://github.com/search?q=org%3Apolkadot-js+abi&type=Code) whereas `metadata` refers to the substrate metadata exposed via RPC.

I think it's more confusing to use two terms for the same thing in different places (metadata & abi) than to just stick to one term (abi) everywhwere.

Reply:
And besides the fact that it's already an established term in the ecosystem, it actually does make sense...

> In computer software, an application binary interface (ABI) is an interface between two binary program modules; often, one of these modules is a library or operating system facility, and the other is a program that is being run by a user.

> An ABI defines how data structures or computational routines are accessed in machine code, which is a low-level, hardware-dependent format; in contrast, an API defines this access in source code, which is a relatively high-level, hardware-independent, often human-readable format. A common aspect of an ABI is the calling convention, which determines how data is provided as input to or read as output from computational routines

Reply:
https://github.com/paritytech/ink/pull/239